src/FSharp.Data.GraphQL.Server.AppInfrastructure/GraphQLWebsocketMiddleware.fs

```diff
@@ -107,7 +107,7 @@ type GraphQLWebSocketMiddleware<'Root>(next : RequestDelegate, applicationLifeti
     task {
       if not (socket.State = WebSocketState.Open) then
         if logger.IsEnabled(LogLevel.Trace) then
-          logger.LogTrace(sprintf "ignoring message to be sent via socket, since its state is not 'Open', but '%A'" socket.State)
+          logger.LogTrace("Ignoring message to be sent via socket, since its state is not 'Open', but '{state}'" socket.State)
```

Comment:
if you do not call `socket.State.ToString()` you do not need to check if the logger is enabled. As there is a check inside

Reply:
Done.